### PR TITLE
travis integrated

### PR DESCRIPTION
**Pull request by shivanggarg1998**

### CODE REVIEW

**Comment by shivanggarg1998** [comment]
#28

File: README.md [hunk 1]
```diff
@@ -52,3 +52,7 @@ This year's edition is developed using Node.js and React and `yarn` is the packa
 
 1. [Vibhor Gupta](https://github.com/VibhorCodecianGupta)
 2. [Gaurav Tiwari](https://github.com/thegauravtiwari)
+
+
+
+[![Build Status](https://travis-ci.com/shivanggarg1998/Bulls-and-Bears.svg?branch=develop)](https://travis-ci.com/shivanggarg1998/Bulls-and-Bears)
```
Comment (VibhorCodecianGupta) [inline README.md]: you added the badge to your repo, change the username to VibhorCodecianGupta

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+services:
+  - mongodb
+before_script:
+  - sleep 15  
+addons:
+  apt:
+    config:
+      retries: true
+    ⋮ # rest (including `packages` and `sources`, if applicable
+
```
Comment (VibhorCodecianGupta) [inline .travis.yml]: Where is the script? what is being executed? I'm guessing you need to check if our tests are running, for that `yarn test` script needs to run on travis

Comment (VibhorCodecianGupta) [reply .travis.yml]: Also, put the badge on the very top, before the main bulls and bears heading

File: README.md (hunk 1 above)
Comment (VibhorCodecianGupta) [reply README.md]: Also, put the badge on the very top, before the main bulls and bears heading

File: .travis.yml (hunk 2 above)
Comment (shivanggarg1998) [reply .travis.yml]: it automatically detects it

Comment (VibhorCodecianGupta) [reply .travis.yml]: No matter, write the script. It's not urgent, take your time. Fix the badge issue urgently though

**Pull request merged**